ios/browser/svg/svg.gni

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+declare_args() {
+  skia_use_fonthost_mac = true
```

Comment:
Is [`//third_party/skia/gn/skia.gni`](https://source.chromium.org/chromium/chromium/src/+/main:third_party/skia/gn/skia.gni) not importable directly?

Reply:
we tried. was having problem with undefined `is_wasm` and other variables. For this svg issue we only need `skia_use_fonthost_mac = true` at this moment in order to able to construct `SkShapers` correctly for iOS.